src/GovUk.Education.ExploreEducationStatistics.Content.Services.Tests/PublicationServiceTests.cs

```diff
@@ -285,9 +290,75 @@ await using (var contentDbContext = InMemoryContentDbContext(contentDbContextId)
 
                 Assert.Equal(publication.Id, publicationViewModel.Id);
                 Assert.False(publicationViewModel.IsSuperseded);
+                Assert.Null(publicationViewModel.SupersededBy);
             }
         }
 
+        [Fact]
+        public async Task Get_IsSuperseded_SupersedingPublicationBuildsSupersededViewModel()
```

Comment:
It confused me for a while when reading the contents of this test but finally worked out it should be named `GetPublicationTree_`... rather than `Get_`... 🙂 

I would also move it much further down the test class to sit below `GetPublicationTree_Superseded` around L1324.

Reply:
Ah yeah, good catch! moved below `GetPublicationTree_Superseded` now